pkg/controller/common/controller_context.go

```diff
@@ -92,6 +95,8 @@ func CreateControllerContext(cb *clients.Builder, stop <-chan struct{}, targetNa
 	configSharedInformer := configinformers.NewSharedInformerFactory(configClient, resyncPeriod()())
 	operatorSharedInformer := operatorinformers.NewSharedInformerFactory(operatorClient, resyncPeriod()())
 
+	imageSharedInformer := imageinformers.NewSharedInformerFactory(imageClient, resyncPeriod()())
```

Comment:
(please ignore if this is annoying :joy:)
```suggestion
	imageSharedInformer := imageinformers.NewSharedInformerFactory(imageClient, resyncPeriod()())
```